GOVERNANCE.md

```diff
@@ -11,21 +11,38 @@ The Fluentd community adheres to the following principles:
 
 ## Voting
 
-The Fluentd project employs "organization voting" to ensure no single organization can dominate the project.
+The Fluentd project employs "organization voting" and "Vetoes" to ensure no single organization can dominate the project.
+
+For formal votes, a specific statement of what is being voted on should be added to the relevant GitHub issue or PR. Maintainers should indicate their yes/no vote on that issue or PR, and after a suitable period of time, the votes will be tallied and the outcome noted.
```

Comment:
```suggestion
For formal votes, a specific statement of what is being voted on should be added to the relevant GitHub issue or PR. Maintainers should indicate their yes/no vote on that issue or PR. After a suitable period of time, the votes will be tallied.
```

It seems to me that this sentence is hard to understand.

I think it would be better to keep it as simple as the following, but I also think it would be better not to mix too many irrelevant modifications.

What do you all think?

Reply:
At least, `and the outcome noted.` shouldn't be removed. This is because the result should be _recorded_.

Reply:
Sorry for my late reply. Thanks for your feedback!

I see! Now I understand what the sentence `the outcome noted` wants to say. It wants to say that `the result should be recorded`.
(It seemed obvious to me that we can see whether the proposal is approved or not. I didn't read the intent, which says we should keep records of the voting results. )

There may be something we can improve, but it would be better to separate it from this fix.